backend/controllers/userController.js

```diff
@@ -4,21 +4,44 @@ module.exports = {
 
     search: async function(req, res) {
 
-        // 활성화 상태의 유저만 조회
-        let query = Object.assign(req.query, { status: 'avail' });
-        let projection = {
-            name: true, rank: true, title: true,
-            group: true, email: true, tel: true
+        const keys = Object.keys(req.query);
+        const valids = ['group', 'name'];
+            
+         // Only allowed fields are Searchable
+        for(let key of keys) {
+            if(!valids.includes(key))
+                throw new BusinessError(`${key} is not allowed param`);
+        }
+
+
+        const projection = {
+            _id: true, name: true, rank: true, status: true,
+            group: true, email: true, tel: true, lastLogin: true
         };
 
         try {
-            const result = await userService.search(query, projection);
+            const result = await userService.find(req.query, projection);
             res.status(200).send(result);
         } catch(err) {
```

Comment:
```javascript
res.status(err.status).send(err.message);
```
this code can occur another error when err.status had not defined.

```javascript
res.status(err.status || 500).send(err.message);
```
this code can send `500` when err.status was not defined.